hugegraph-test/src/main/java/com/baidu/hugegraph/core/EdgeLabelCoreTest.java

```diff
@@ -873,6 +875,65 @@ public void testRemoveEdgeLabelWithEdgeAndSecondaryIndex() {
         });
     }
 
+    @Test
+    public void testRebuildIndexOfEdgeLabelWithoutLabelIndex() {
+        Assume.assumeFalse("Support query by label",
+                           storeFeatures().supportsQueryByLabel());
+
+        initDataWithoutLabelIndex();
+
+        // Not support query by label
+        Assert.assertThrows(NoIndexException.class, () -> {
+            graph().traversal().E().hasLabel("read").toList();
+        }, e -> {
+            Assert.assertTrue(
+                   e.getMessage().startsWith("Don't accept query by label") &&
+                   e.getMessage().endsWith("it disables label index"));
```

Comment:
use assertContains() instead and rebase on https://github.com/hugegraph/hugegraph-common/pull/43